<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrudKabupaten</class>
 <widget class="QMainWindow" name="CrudKabupaten">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manajemen Data Kabupaten</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <item>
     <widget class="QLabel" name="labelJudul">
      <property name="text">
       <string>Data Kabupaten</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="cariLayout">
      <item>
       <widget class="QLabel" name="labelCari">
        <property name="text">
         <string>Cari</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputCari"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableKabupaten"/>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QPushButton" name="btnTambah">
        <property name="text">
         <string>Tambah</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEdit">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnHapus">
        <property name="text">
         <string>Hapus</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPrint">
        <property name="text">
         <string>Print</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
